psi4/driver/frac.py

```diff
@@ -481,49 +582,45 @@ def ip_fitting(molecule, omega_l, omega_r, **kwargs):
     types = []
 
     # Right endpoint
-    core.set_global_option('DFT_OMEGA', omega_r)
+    core.set_local_option('SCF', 'DFT_OMEGA', omega_r)
 
     # Neutral
     if read:
-        core.set_global_option("GUESS", "READ")
+        core.set_local_option("SCF", "GUESS", "READ")
         p4util.copy_file_to_scratch(read180, 'psi', 'ot', 180)
 
     molecule.set_molecular_charge(charge0)
     molecule.set_multiplicity(mult0)
-    core.print_out("""\n\t==> IP Fitting SCF: Neutral, Right Endpoint <==\n""")
-    E0r, wfn = energy('scf', return_wfn=True, molecule=molecule, **kwargs)
+    E0r, wfn = driver.energy('scf', dft_functional=name, return_wfn=True, molecule=molecule,
+                             banner='IP Fitting SCF: Neutral, Right Endpoint', **kwargs)
```

Comment:
+1 for the banner feature!

Reply:
Indeed, I think the banner looks far tidier. (and more noticable)